src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java

```diff
@@ -84,6 +99,25 @@ void should_return_bad_request_when_form_type_is_invalid() throws Exception {
             .andExpect(status().isBadRequest());
     }
 
+    @Test
+    void should_ignore_unrecognised_fields_and_return_success_message() throws Exception {
```

Comment:
This is a valid test case for the validator, but as the validator is mocked here and the controller only cares about the outcome (success/failure), I think we can remove this test. Unless I'm missing something.

Reply:
Moved this test to Validtor test class